app/lib/project/OTFExportController.js

```diff
@@ -0,0 +1,95 @@
+define([
+    'metapolator/errors'
+  , 'metapolator/rendering/glyphBasics'
+  , 'metapolator/rendering/OpenTypePen'
+  , 'ufojs/tools/pens/PointToSegmentPen'
+  , 'opentype'
+  , 'metapolator/models/MOM/Glyph'
+  , 'metapolator/timer'
+], function(
+    errors
+  , glyphBasics
+  , OpenTypePen
+  , PointToSegmentPen
+  , opentype
+  , MOMGlyph
+  , timer
+) {
+    "use strict";
+
+    function OTFExportController(master, model, masterName) {
+        this._master = master;
+        this._model = model;
+        this._masterName = masterName;
+    }
+    var _p = OTFExportController.prototype;
+
+    _p.do_export = function() {
+        var glyphs = this._master.children
+          , glyph
+          , updatedUFOData
+          , i, l, v, ki, kil, k, keys
+          , style
+          , time, one, total = 0
+          , font
+          , otf_glyphs = []
+          , renderer = {
+                  penstroke: glyphBasics.renderPenstrokeOutline
+                , contour: glyphBasics.renderContour
+            }
+          ;
+
+        console.warn('exporting OTF ...');
+        for(i = 0,l=glyphs.length;i<l;i++) {
+            var otPen = new OpenTypePen()
+              , pen = new PointToSegmentPen(otPen)
+              ;
+
+            glyph = glyphs[i];
+            style = this._model.getComputedStyle(glyph);
+            time = timer.now();
+
+            // Allow the glyph ufo data to be updated by the CPS.
+            updatedUFOData = glyph.getUFOData();
+            keys = Object.keys(updatedUFOData);
+            for(ki=0,kil=keys.length;ki<kil;ki++) {
+                try {
+                    k = keys[ki];
+                    v = style.get(MOMGlyph.convertUFOtoCPSKey(k));
+                    updatedUFOData[k] = v;
+                }
+                catch( error ) {
+                    if(!(error instanceof errors.Key)) {
+                        throw error;
+                    }
+                }
+            }
+
+            glyphBasics.drawGlyphToPointPen ( renderer, this._model, glyph, pen );
+
+            otf_glyphs.push(new opentype.Glyph({
```

Comment:
Hmm, I think we can savely consider all the xMin, xMax, yMin, yMax values Bugs. They are simply wrong, none of these represents the real glyph min/max values.

If I merge this we can file the bug on the way.

Memo: The best way would be to port: https://github.com/robofab-developers/robofab/blob/python3k/Lib/robofab/pens/boundsPen.py this module to `ufoJS/lib/pens` maybe using http://pomax.github.io/bezierjs/ instead of porting the robofab math modules.